indexing-service/src/main/java/org/apache/druid/indexing/common/task/DefaultTaskIdentitiesProvider.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.indexing.common.task;
+
+import com.google.common.base.Strings;
+import org.apache.druid.query.DruidMetrics;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class DefaultTaskIdentitiesProvider implements TaskIdentitiesProvider
+{
+  public static final String TYPE = "default";
+
+  @Override
+  public Map<String, Object> getTaskMetricTags(Task task)
+  {
+    String taskIdentifier = task.getType();
```

Comment:
Should we consider adding a `groupType` to the task object? Parsing the group IDs name is pretty fragile.

Reply:
That's a valuable suggestion, however, implementing it could require modifications across numerous task types. I'd like to keep the scope of this PR focused, and consider that as potential follow-up work.